tests/test_reload.py

```diff
@@ -19,6 +19,7 @@
 
 class C(pg.QtCore.QObject):
     sig = pg.QtCore.Signal()
+    @pg.QtCore.Slot()
```

Comment:
If you remove this, would you still get the segfault?

Reply:
No - that is exactly the part that triggers it.